src/jfrMetadata.h

```diff
@@ -236,8 +229,12 @@ class JfrMetadata : Element {
         return &_root;
     }
 
-    static std::vector<std::string>& strings() {
-        return _strings;
+    static void forEachStringOrdered(std::function<void(const std::string&)> consumer) {
+        _strings.forEachOrdered(consumer);
+    }
+
+    static size_t stringsSize() {
+        return _strings.size();
```

Comment:
Maybe just expose `_strings`? These `forEachStringOrdered` and `stringsSize` look too artificial.

Reply:
b9785616cb1dc262d64635206629cce317c2f162

Reply:
I meant both `forEachStringOrdered` and `stringsSize` could be replaced with `strings()`